helm-charts/0.21.5/values.yaml

```diff
@@ -0,0 +1,140 @@
+# /*
+# |    Protect your secrets, protect your sensitive data.
+# :    Explore VMware Secrets Manager docs at https://vsecm.com/
+# </
+# <>/  keep your secrets… secret
+# >/
+# <>/' Copyright 2023–present VMware, Inc.
+# >/'  SPDX-License-Identifier: BSD-2-Clause
+# */
+
+global:
+  deploySpire: true
+  # possible options for baseImage (distroless, distroless-fips, photon, photon-fips)
+  baseImage: distroless
+  registry: vsecm
+  logLevel: "7"
+  images:
+    safe:
+      distrolessRepository: vsecm-ist-safe
+      distrolessFipsRepository: vsecm-ist-fips-safe
+      photonRepository: vsecm-photon-safe
+      photonFipsRepository: vsecm-photon-fips-safe
+      tag: 0.21.4
+      pullPolicy: IfNotPresent
+    sentinel:
+      distrolessRepository: vsecm-ist-sentinel
+      distrolessFipsRepository: vsecm-ist-fips-sentinel
+      photonRepository: vsecm-photon-sentinel
+      photonFipsRepository: vsecm-photon-fips-sentinel
+      tag: 0.21.4
+      pullPolicy: IfNotPresent
+    spireAgent:
+      repository: ghcr.io/spiffe/spire-agent
+      tag: 1.8.0
+      pullPolicy: IfNotPresent
+    spiffeCsiDriver:
+      repository: ghcr.io/spiffe/spiffe-csi-driver
+      tag: 0.2.3
+      pullPolicy: IfNotPresent
+    nodeDriverRegistrar:
+      repository: registry.k8s.io/sig-storage/csi-node-driver-registrar
+      tag: v2.8.0
+      pullPolicy: IfNotPresent
+    spireServer:
+      repository: ghcr.io/spiffe/spire-server
+      tag: 1.8.0
+      pullPolicy: IfNotPresent
+    spireControllerManager:
+      repository: ghcr.io/spiffe/spire-controller-manager
+      tag: 0.3.0
+      pullPolicy: IfNotPresent
+  vsecm:
+    namespace: vsecm-system
+  spire:
+    namespace: spire-system
+    logLevel: DEBUG
+    serverPort: 8081
+
+
+podAnnotations:
+  "helm.sh/hook": pre-install
+
+replicaCount: 1
+
+image:
+  repository: nginx
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
```

Comment:
I’d recommend cleaning the attributes that we don’t use for simplicity and clarity.

Reply:
+1, will do. Thanks!! 👍